templates/index.php

```diff
@@ -97,3 +96,4 @@ class="icon-loading"
             news-scroll-enabled-mark-read="Content.markReadEnabled()"
             news-scroll-auto-page="Content.autoPage()"
             news-scroll-mark-read="Content.scrollRead(itemIds)"></div>
+
```

Comment:
This file seems to be changed accidentally.

Reply:
```suggestion
```